docs/orchestrate/saucectl-configuration.md

```diff
@@ -0,0 +1,164 @@
+:::
+
+### `apiVersion`
+
+<p><small>| REQUIRED | STRING |</small></p>
+
+Identifies the version of the underlying configuration schema. At this time, `v1alpha` is the only supported value.
+
+```yaml
+apiVersion: v1alpha
+```
+
+### `kind`
+
+<p><small>| REQUIRED | STRING |</small></p>
+
+Tells SauceCTL this is a Sauce Orchestrate configuration. `imagerunner` is the required value for Sauce Orchestrate.
+
+```yaml
+kind: imagerunner
+```
+
+### `workload`
+
+<p><small>| REQUIRED | STRING |</small></p>
+
+Tell Sauce Orchestrate what kind of entrypoint process you will running. This is important for security monitoring. Options are `webdriver` or `other`. For most configurations `webdriver` will be required.
+
+```yaml
+kind: imagerunner
+```
+
+### `image`
+
+<p><small>| REQUIRED | STRING |</small></p>
+
+The location of your Docker image. Takes the format [registry]/[image]:[tag].
+
+```yaml
+image: saucelabs/sl-demo-docker-primary:0.0.1
+```
+
+### `imagePullAuth`
+
+<p><small>| OPTIONAL | OBJECT |</small></p>
+
+The credentials needed to access an image hosted in a private registry. It is highly recommend to not hardcode credentials in your SauceCTL cnofig. Use environment variables instead.
+
+```yaml
+imagePullAuth:
+  user: $DOCKER_USERNAME
+  token: $DOCKER_PASSWORD
+```
+
+### `entrypoint`
+
+<p><small>| REQUIRED | STRING |</small></p>
+
+The command that is executed once the container is ready.
+
+```yaml
+entrypoint: mvn test
+```
+
+### `files`
+
+<p><small>| OPTIONAL | ARRAY |</small></p>
+
+Files to be uploaded onto the container. Useful for populating test data that your scripts access. src and dst must be an absolute path.
+
+```yaml
+files:
+  - src: "runsauce.json"
+    dst: "/workdir/runsauce.json"
+```
+
+### `env`
+
+<p><small>| OPTIONAL | ARRAY |</small></p>
+
+Environment variables to be injected into the container. Useful for populating secrets used in your tests. These are not stored anywhere in Sauce Labs.
+
+```yaml
+env:
+  KEY: value
+```
+
+### `artifacts`
+
+<p><small>| OPTIONAL | ARRAY |</small></p>
+
+In order for SauceCTL to download files from the Sauce Orchestrate container two configurations are needed. The first is to tell Orchestrate which files to upload from the container once your entrypoint command finishes.
+
+```yaml
+# declared within your suites definition
+suites:
+  ...
+  artifacts:
+    - "/workdir/best-practice/target/surefire-reports/*"
+```
+
+Then you must tell SauceCTL to download the artifacts to your machine.
+
+```yaml
+# declared at the top level of your confi.yml
+artifacts:
+  download:
+    when: always
+    match:
+      - "*"
+    directory: ./artifacts
+```
+
+:::note
+The following limitations are in effect:
+
+- Must specify an absolute path (starting at the root `/`)
+- Max requested volume for parent dir is 10M
+- You can only specify up to 10 paths
```

Comment:
Maybe we should mention that this applies only to `files` not `artifacts` downloads ?